utils.go

```diff
@@ -0,0 +1,48 @@
+package forseti
+import (
+	"strconv"
+	"math"
+	"strings"
+)
+
+func isTypeValid(strType string) bool {
+	list := []string {"BIKE", "SCOOTER", "MOTORSCOOTER", "STATION", "CAR", "OTHER"}
+	for _, b := range list {
+		if strings.EqualFold(strType, b) {
+			return true
+		}
+    }
+    return false
+}
+
+func stringToInt(inputStr string, defaultValue int) int {
+	input, err := strconv.Atoi(inputStr)
+	if err != nil {
+		input = defaultValue
+	}
+	return input
+}
+
+func hsin(theta float64) float64 {
+	return math.Pow(math.Sin(theta/2), 2)
+}
+
+func coordDistance(lat1, lon1, lat2, lon2 float64) float64 {
```

Comment:
We could make the interface a little simpler to use :) 
```suggestion
func coordDistance(from, to Coord) float64 {
```